[CQA] Shorten abstracts in Provisioning Hosts (#4954)
theforeman/foreman-documentation: guides/common/modules/proc_preparing-a-synchronized-kickstart-repository.adoc

= Preparing a synchronized Kickstart repository

[role="_abstract"]

> You can use synchronized Kickstart repositories to install the operating system on provisioned hosts.

Reviewer: ```suggestion
You can use synchronized Kickstart repositories as the source of the operating system during host provisioning.
```

IMO once the host is provisioned, it _has_ an OS installed and does not really need the Kickstart repos anymore.

Author: Agree, but would like to check with @stejskalleos . Is this accurate?

Reviewer 2: I agree with the wording by Max. To me, it looks more accurate.

Reviewer 3: LGTM
